webproject/src/main/java/com/google/sps/data/Room.java

```diff
@@ -46,14 +49,24 @@ private Room (Builder builder) {
     public double getMinPrice() {return minPrice;}
 
     public double getDeliveryFee() {return deliveryFee;}
+
     public List<String> getUsers() {return users;}
 
     public String getCreator() {return creator;}
 
+    public List<Order> getOrders() {return orders;}
+
+    public double getOrdersValue() {return ordersValue;}
```

Comment:
For this, the order value is by the room, so I thought it might be easier to get the value from the room data itself

Reply:
If we have a separate document for orders, it will be hard to query the ordersValue of a room and do the validation of the room to check if ordersValue is more than the minimum price

Reply:
right. I think we can keep i this way, but we need to implement trigger
https://firebase.google.com/docs/functions/database-events
to run a function to update ordersvalue whenever theres any edit/insert/delete in the orders document